rep-0152.rst

```diff
@@ -0,0 +1,142 @@
+Being a maintainer is not an easy job.
+It includes responding to incoming questions and issues, checking for proposed changes and deciding to add them to the main source tree, and other responsibilities.
+
+The idea behind this document is to organize a team of volunteers who can do ONLY the release job, which includes sending the source tree to the ROS Build Farm and acting on notification emails form the builds.
+The volunteers of the release maintainers group will enable the community to continue using the same set of packages from previous ROS distros with minimal community effort.
+To this end, we followed the Debian QA Group system and the Debian Developer's Reference ('Dealing with inactive and/or unreachable maintainers'[3]_), which is considered non-normative, but best practice.
+
+In ROS, we use 'Unmaintained' package status([4]_) and ROS Orphaned Packages Group([5]_), who do not have to take full responsibility for package maintenance jobs but are expected to take minimum responsibilities which mostly include releasing their packages into newer ROS distros.
+
+Reference
+=========
+
+REP-0140 [6]_ and REP-0141 [7]_ provide maintainer information in package manifest file and maintainer status in ROS distribution file.
+
+Maintenance Guide in ROSWiki [8]_ provides how to claim maintainership for orphaned packages.
+
+Operations
+==========
+
+1-A. Orphaning a package by maintainers
+---------------------------------------
+
+If you can no longer maintain your package.
+
+a. Set the package maintainer section in the ``package.xml`` file as follows.
+
+   ``<maintainer email="ros-orphaned-packages@googlegroups.com">ROS Orphaned Package Maintainers</maintainer>``
+
+b. Post a message to the `ROS Discourse Release Category <http://discourse.ros.org/c/release>`_ indicating that package is now orphaned.
+   Wait for response from someone in the ROS Orphaned Package Maintainer team.
+
+c. Add admin permission of both source and release repository to who respond from in the ROS Orphaned Package Maintainers team.
+
+1-B. Orphaning a package by users
+---------------------------------
+
+If you find a package is not actively maintained and are willing to release as a ROS Orphaned Package Maintainer.
+
+a. Contact the current maintainer, ask for a transition to orphaned package and request access to the source and release repository to make releases.
+
+b. If the current maintainer is not responsive (a single day isn't enough to conclude that) and you still want to release the packages, there are two different paths forward depending on the "location" of the source and release repositories.
+
+   1. If the repositories are hosted on GitHub within a `ros-` org unit, the admins of the org group will usually grant the necessary access to these repositories.
```

Comment:
This is what gets done in the end, but it doesn't tell the interested party what to do.  In other words, I think we need a decision tree here which is something like:

* Contact the maintainer.
* Maintainer responded? ----> done
* Maintainer didn't respond after 3 months?
    * Send information to ros-orphaned-packages@googlegroups.com about the source repository, release repository, and where and how maintainer failed to respond.
    * If the ROS Orphaned Package Maintainers agree with your assessment, and the repository is hosted on GitHub within a `ros-` organization, the admins of the group can grant the necessary access.
    * If no one aside from the original author has access, you should fork the repository

Reply:
thanks for the comment, I have uploaded a fixed version. As for the last comment, do you mean we'd add something like
![diagram](https://user-images.githubusercontent.com/493276/108445992-b875e500-72a0-11eb-875f-82d51677dd42.png) ???

Reply:
Yeah, that is kind of the thing I was thinking of.  Basically a clear set of steps that a person needs to go through in order to take over a package.

Reply:
@clalancette please check the diagram at https://github.com/ros-infrastructure/rep/pull/150#discussion_r587199922, which includes all processes.